plantuml-parser-core/src/test/java/com/shuzijun/plantumlparser/core/UmlGenerationTester.java

```diff
@@ -0,0 +1,120 @@
+package com.shuzijun.plantumlparser.core;
+
+import com.github.javaparser.ParserConfiguration;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class UmlGenerationTester {
+
+    private static ParserConfig makeParserConfig(String filename, boolean showDefaultCon) {
```

Comment:
If you copied code from `TestRecord.java`, you should have `TestRecord` call this code, rather than duplicate it.

Reply:
My intention was that UmlGenerationTester replaces both TestRecord.java and TestDefaultConstructor.java. My plan was to delete these files, but I left them until I get UmlGenerationTester approved by you. What do you think about this?

Reply:
Good to know. Because old files are never truly lost, you can delete files casually in git.